### PR TITLE
[UIE-181] Figma integration into Storybook

### PR DESCRIPTION
**Pull request by cahrens**
### Jira Ticket: https://broadworkbench.atlassian.net/browse/UIE-181

Unlike our previous story integration with Figma, which only displayed within the Chromium view, this Storybook plugin allows integration directly in the Storybook documentation.

Here is the branch Storybook instance: https://65fc89c9335768720ff8605a-zirepihlgp.chromatic.com/?path=/story/packages-components-buttonprimary--primary

And a screenshot of an example:
image

### CODE REVIEW

**Review by nawatts — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

File: src/stories/packages/components/ButtonOutline.stories.ts [hunk 1]
```diff
@@ -46,9 +46,16 @@ const meta: Meta<typeof ButtonOutline> = {
...
 type Story = StoryObj<typeof ButtonOutline>;
 
-export const Primary: Story = {
+export const Outline: Story = {
   args: {
     tooltip: 'This can provide additional context',
     children: 'Cancel',
   },
+  parameters: {
+    design: {
```
Comment (msilva-broad) [inline src/stories/packages/components/ButtonOutline.stories.ts:55]: can we list more than one figma link per component or in other storybook documentation blocks?

File: .storybook/main.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ const config: StorybookConfig = {
     getAbsolutePath('@storybook/addon-onboarding'),
     getAbsolutePath('@storybook/addon-interactions'),
     getAbsolutePath('@storybook/addon-a11y'),
+    getAbsolutePath('@storybook/addon-designs'),
```
Comment (msilva-broad) [inline .storybook/main.ts:28]: Where is the old plugin being removed?  Or is that planned for a followup PR?

File: src/stories/packages/components/ButtonOutline.stories.ts (hunk 1 above)
Comment (cahrens) [reply src/stories/packages/components/ButtonOutline.stories.ts:55]: I haven't seen an example of that in their documentation, but it's something we can explore.

File: .storybook/main.ts (hunk 2 above)
Comment (cahrens) [reply .storybook/main.ts:28]: The other one wasn't a plugin-- it's part of Chromatic, and I don't think we did anything to enable it.

File: src/stories/packages/components/ButtonOutline.stories.ts (hunk 1 above)
Comment (cahrens) [reply src/stories/packages/components/ButtonOutline.stories.ts:55]: Ah, it turns out that you can pass an array to design! I'll let Callie know about that.

image

image

**Pull request merged**